Lib/__future__.py

```diff
@@ -143,5 +143,5 @@ def __repr__(self):
                           CO_FUTURE_GENERATOR_STOP)
 
 annotations = _Feature((3, 7, 0, "beta", 1),
-                       (3, 11, 0, "alpha", 0),
+                       None,
```

Comment:
An adjustment to the module docstring extending the cases where `mandatoryRelease` can be `None` would not be amiss.

Reply:
It already states that:
> MandatoryRelease may also be None, meaning that a planned feature got
dropped.

Reply:
Yeah, but this feature hasn't been dropped yet. It's just not decided when it will become the default.

Reply:
Done